server/src/games/Games.TyrantsOfTheUnderdark/TyrantsOfTheUnderdarkScoringModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace BGS.Games.TyrantsOfTheUnderdark;
+
+internal record TyrantsOfTheUnderdarkScoringModel
+{
+    public byte TrophyHall { get; init; }
+    
+    public byte Deck { get; init; }
+    
+    public byte InnerCircleDeck { get; init; }
+    
+    public byte Tokens { get; init; }
+    
+    public byte ControlSites { get; init; }
+    
+    public byte TotalControlSites { get; init; }
```

Comment:
Let's sort and name it like a rule book:

- The VP value of each site you control (ControlSites rename to **ControlSitesValue**);
- 2 VP for each site under your total control (TotalControlSites rename to **CountOfTotalControlSites**);
- 1 VP for each troop in your trophy hall (TrophyHall rename to **CountOfTroopInTrophyHall**);
- The deck VP value for each card in your deck, hand, and discard pile (Deck rename to **DeckValue**);
- The inner-circle VP value for each card in your inner circle (InnerCircleDeck rename to **InnerCircleValue**);
- VP tokens gained during the game (Tokens rename to **VictoryPointTokens**).

P.S. don't forget rename [in test](https://github.com/TexnikBy/BGS/pull/20/files#diff-5647a151939579275932fe1e574ff915a0de59bc940d8f1ad4167252040c13dbR33-R38)